src/PanoViewer/PanoViewer.js

```diff
@@ -257,7 +256,8 @@ export default class PanoViewer extends Component {
 
 	_bindRendererHandler() {
 		this._photoSphereRenderer.on(PanoImageRenderer.EVENTS.IMAGE_LOADED, e => {
-			this.trigger(EVENTS.CONTENT_LOADED, e);
+			this._activate();
+			this.trigger(EVENTS.READY, e);
```

Comment:
@happyhj Difference between which I've commit on (https://github.com/naver/egjs-view360/pull/157) is that `READY` event is triggering before `startRender` or not. Did you mean it?

Although ready event is triggered before `startRender`, we can guarantee PanoViewer handle user interaction. Right?